sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-part-time-process-response-integration.scheduler.e2e-spec.ts

```diff
@@ -87,9 +88,14 @@ describe(
         },
         { msfaaNumber: THROW_AWAY_MSFAA_NUMBER },
       );
+      // Update the date sent for the notifications to current date where the date sent is null.
+      await db.notification.update(
+        { dateSent: IsNull() },
+        { dateSent: new Date() },
+      );
     });
 
-    it("Should process an MSFAA response with confirmations and a cancellation and update all records when the file is received as expected.", async () => {
+    it("Should process an MSFAA response with confirmations and a cancellation and update all records when the file is received as expected. MSFAA cancellation notification should also be persisted.", async () => {
```

Comment:
Can we have it adjusted as below?
"Should process an MSFAA response with confirmations and cancellations, and save a notification message when the file is received as expected.